src/backend/user_files.py

```diff
@@ -289,6 +289,7 @@ def alt_or_cmd() -> str:
             "theme": "Midnight.qpl",
             "writeAliases": {},
             "zoom": 1.0,
+            "roomBookmarkIDs": [],
```

Comment:
```suggestion
            "roomBookmarkIDs": {},
```
Related to the above suggestion, to have this setting be a dict of `{user_id: [room_id, ...]}`